grid.py

```diff
@@ -228,22 +228,37 @@ def populate_grid(self):
         self.populate_columns()
 
     def lines_connected(self):
+        """
+        Checks whether all lines in the crossword are connected, and that no breaks exist.
```

Comment:
```suggestion
        Checks whether all lines in the crossword are connected, and that no breaks exist.
        Returns:
        bool: `True` if all lines are connected and no breaks exist, `False` otherwise.
```